examples/gno.land/p/demo/zteams/errors.gno

```diff
@@ -0,0 +1,13 @@
+package zteams
+
+import "errors"
+
+var (
+	ErrNotOwner               = errors.New("not the owner")
+	ErrNotAuthorized          = errors.New("not authorized")
+	ErrUserNotDisburser		  = errors.New("cannot disburse")
+	ErrTeamNotFound           = errors.New("team not found")
+	ErrUserNotMember          = errors.New("user not a member")
+	ErrUserAlreadyMember      = errors.New("user is already a member")
+	ErrTeamAddressAlreadyExists = errors.New("team address already exists")
+)
```

Comment:
Are these error types intended to be used in other packages? If not, I think it would be better to change them to private.

Reply:
Changed errors types to private.